src/ImageSharp/PixelFormats/Generated/PixelOperations{TPixel}.Generated.tt

```diff
@@ -120,6 +162,9 @@ namespace SixLabors.ImageSharp.PixelFormats
     GeneratePackFromMethodUsingPackFromRgba32("Bgr24", "rgba.Bgr = Unsafe.Add(ref sourceRef, i);");
     GenerateToDestFormatMethods("Bgr24");
 
+    GeneratePackFromMethodUsingPackFromArgb32("Argb32", "argb = Unsafe.Add(ref sourceRef, i);");
```

Comment:
Are we not missing the method generation here for `Rgba32`, `Rgb24`, `Bgra32`, and `Bgr24`?

Reply:
I played with it a bit, but I think I'm getting method signature clashes between the Argb32 and Rgba32 versions because the method sigs are the same.

Reply:
No worries, I'm cloning your fork now so will have a look